server/controllers/tasksController.test.ts

```diff
@@ -1,15 +1,18 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I would prefer if we did this on a per-line basis with: 
` // eslint-disable-next-line @typescript-eslint/no-explicit-any` above the specific line we need it